Improve logging and documentation about objcopy dependency (#13582)
quarkusio/quarkus: docs/src/main/asciidoc/building-native-image.adoc

add `-Dquarkus.native.debug.enabled=true` flag when generating the native executable.
You will find the debug symbols for the native executable in a `.debug` file next to the native executable.

[NOTE]
====

> The generation of the `.debug` file requires `objcopy` being present in the `PATH` environment variable.

Reviewer: ```suggestion
The generation of the `.debug` file requires `objcopy` being available from the `PATH` environment variable.
```
The way it is written, you have the impression you need to add `objcopy` to PATH.

Author: I changed this part to include some more info on how to get `objcopy` as well.
